README.md

```diff
@@ -84,6 +83,62 @@ python run.py
 deactivate
 ```
 
+### Run app in Linux
+
+The project runs on Python 3.
+
+1. Create a virtual enviorntment:
+`$ virtualenv venv`
```

Comment:
```suggestion
`virtualenv venv`
```
for consistency, and some beginners may not understand that "$" is not part of the command to be run on terminal